### PR TITLE
ui:ssl setup using env variables and docker build args

### LINKED ISSUE #418: ui: Simplify installation of ssl certificates
### Description
To enable https on Trubudget certificates have to be installed. No matter how the certificates are generated, enabling https in the Trubudget frontend is a complex and uncomfortable process. This should be simplified.

### Suggestion
Pass an env variable to the frontend where the certificates can be found. The rest of the process shall be automatically done. 

#### Current way of enabling https

- expose port 443 in the frontend container
- adjust the nginx.conf
```example nginx
# add https redirect
server {
    listen         80;
    server_name    my.domain.net;
    return         301 https://my.domain.net$request_uri;
}
# ssl
server {
    listen       443 ssl default_server;
    server_name  my.domain.net;
    ssl_certificate     /etc/ssl/my.domain.net/fullchain.pem;
    ssl_certificate_key /etc/ssl/my.domain.net/privkey.pem;
    [...]
}
```
- copy certificates into container

### PR DESCRIPTION
**Pull request by galethil**
### Checklist

- [x] I have followed (at least) the [PR section of the contributing guide](https://github.com/openkfw/TruBudget/blob/master/CONTRIBUTING.md).

### Description

This PR adds option to setup secure HTTPS connection on nginx using environment variables passed to docker.
SSL certificates had to be added to specific folder and are not committed to Git for security reasons.
In case of SSL nginx config is generated from template.

Closes #418

### CODE REVIEW

File: frontend/certificates/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Certificates
+
+Copy your certificates to this folder. Two files are expected, both with *.pem* extension.
+
+In order to use certificates to establish secure HTTPS generate certificate and private key files in .pem format. Default names are **certificate.pem** and **privatekey.pem**.
+
+If you want to use HTTPS and you have copied certificates you need to specify 2 build arguments for frontend Dockerfile:
```
Comment (SamuelPull) [inline frontend/certificates/README.md:7]: @galethil  These are ENV, not ARG, so perhaps specify environment variables?

**Comment by SamuelPull** [comment]
@galethil It might be good idea to have docker yml files and scripts setup, perhaps dummy certificates, for testing and development, set up.

File: frontend/certificates/README.md (hunk 1 above)
Comment (galethil) [reply frontend/certificates/README.md]: @SamuelPull I changed description to "environment variables"

**Comment by SamuelPull** [comment]
Documented additional necessary prerequisites for setting up ssl termination reverse proxy, and pushed the changes in readme along with other files.

**Review by SamuelPull — APPROVED** [review]

**Pull request merged**